piperider_cli/workspace.py

```diff
@@ -748,7 +749,7 @@ def _run_dbt_command(table, default_schema, dbt, console):
         return
 
     cmd = dbt.get('cmd', 'test')
-    if cmd not in ['build', 'run', 'test']:
+    if cmd not in ['build', 'test']:
         message = f"'dbt {cmd}' is invalid, only support 'dbt build/run/test'."
```

Comment:
We should change the warning message as well. `build/run/test` -> `build/test`

Reply:
Or we could remove L751-757 because we check dbt command outside

Reply:
Agree, remove it.